server/controllers/authentication/validateEmail.ts

```diff
@@ -0,0 +1,30 @@
+import { Request, Response } from 'express';
+import dotenv from 'dotenv';
+import { findAccount, updateValidate } from '../../database/queries/signUpUser';
+
+dotenv.config();
+
+const validateEmail = async (req: Request, res: Response) => {
+  try {
+    const { accessToken }: { accessToken?: string } = req.query;
```

Comment:
You need to verify the accessToken first, and why do you store it in the database? you already sent the email of this user with the token